debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -202,6 +202,12 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
         };
     }
 
+    private static inferVolumes(folder: WorkspaceFolder, debugConfiguration: DockerDebugConfiguration): DockerContainerVolume[] {
+        return debugConfiguration && debugConfiguration.dockerRun && debugConfiguration.dockerRun.volumes
+            ? debugConfiguration.dockerRun.volumes.map(volume => ({ ...volume, localPath: DockerDebugConfigurationProvider.resolveFolderPath(volume.localPath, folder) }))
```

Comment:
Just curious: Did you find the spread operator more readable than Javascript's `Object.assign(volume, {localPath: resolveFoderpath})`?

Reply:
Personally I find spread much easier to read.

Reply:
Yes, `Object.assign()` has always felt clunky to me.  We used it extensively on other TS projects before the spread operator was available; I'm happy it finally made it into the language.

Reply:
Sounds good. We've used Object.assign() before, so I was wondering if we should use this convention moving forward.